tests/Nhs.Appointments.Api.Integration/Nhs.Appointments.Api.Integration.csproj

```diff
@@ -41,4 +41,8 @@
 	  <None Remove="Nhs.Appointments.Api.Integration\**" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Remove="Scenarios\Availability\DailyAvailability.feature" />
+	</ItemGroup>
```

Comment:
Is this necessary?

Reply:
Wouldn't have thought so, it was added automatically by the looks of it. I'll remove it.